admiral/pkg/clusters/virtualservice_routing.go

```diff
@@ -1493,3 +1554,202 @@ func performInVSRoutingRollback(
 	}
 	return nil
 }
+
+func mergeVS(
```

Comment:
can we have a happy path test for this methods, so that we have one end-to-end usecase?